packages/@dcl/ecs/tsconfig.cjs.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "module": "CommonJS",
+    "outDir": "dist-cjs"
```

Comment:
maybe `dist/cjs` ?

Reply:
I'll try it, I guessed there could be some problems, but I didn't tried

Reply:
it is better to not mix two module systems inside `dist`, i'd keep dist-cjs